ldap/servers/slapd/ldapi.c

```diff
@@ -0,0 +1,409 @@
+/** BEGIN COPYRIGHT BLOCK
+ * Copyright (C) 2020 Red Hat, Inc.
+ * All rights reserved.
+ *
+ * License: GPL (version 3 or any later version).
+ * See LICENSE for details.
+ * END COPYRIGHT BLOCK **/
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include <unistd.h>
+#include "slap.h"
+
+#if defined(ENABLE_LDAPI)
```

Comment:
Could be above the includes as we don't need anything unless ldapi is enabled (but also, we want to remove this define so probably not a nitpick that matters ....)

Reply:
I will take care of the defines in a separate ticket.